tests/functional/CoreTest.php

```diff
@@ -40,23 +40,47 @@ public function testSiteSelectionViaCwd()
 
     public function testOptionsUri()
     {
-        // Put a yml file in the drush folder.
+        // Test whether a URI in a config file resolves correctly, and test
+        // various URI values for their expected Site URI and path.
         $drush_config_file = Path::join($this->webrootSlashDrush(), 'drush.yml');
-        $test_uri = 'http://test.uri';
```

Comment:
We still want to test with `--uri=http://test.uri`. Looks like you removed this test.

Reply:
Two answers, to make surer I'm not misinterpreting:
* This test _is_ still being done; it's just done as part of the loop.
* This (the current/old test) isn't testing with --uri=http://test.uri; it's only writing "http://test.uri" to the config file and testing with --uri=OMIT on the commandline. It's slightly hard to see because of naming, that $options_with_uri is not the --uri option on the commandline. Which is also why I renamed the variables.

Reply:
I could _add a new_ drush invocation that tests --uri on the commandline (not writing config fille), though. Either with only one flag, or with all of them.

Reply:
I'll have to re-read it later in context, but I think that `$options_with_uri` *does* have the `--uri` option on the commandline. At least that was my impression.

Reply:
Ah, I see now. Tests look good.